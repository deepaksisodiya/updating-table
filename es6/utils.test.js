const updateCurrencyPairsMidPrices = require('./utils');


describe('updateCurrencyPairsMidPrices', () => {
  it('calls first time', () => {
    const data = {
      "bestBid": 2,
      "bestAsk": 4,
      "name": "usdjpy",
    };
    const allCurrencyPairsMidPrices = {};
    const time = 132345321;

    updateCurrencyPairsMidPrices(data, allCurrencyPairsMidPrices, time);

    expect(allCurrencyPairsMidPrices).toMatchSnapshot();
  });
});
