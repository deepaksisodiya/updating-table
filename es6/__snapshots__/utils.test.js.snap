// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`updateCurrencyPairsMidPrices calls first time 1`] = `
Object {
  "usdjpy": Array [
    Object {
      "midPrice": 3,
      "time": 132345321,
    },
  ],
}
`;
